### PR TITLE
Chore: Added web-resources-project in opensource projects

### LINKED ISSUE #575: Add web-resources-project in opensource projects
### Site URL

https://github.com/jayk-gupta/web-resources-project

### Description of the above provided site

The project aims to provide developers with a centralized platform to access a wide range of frontend tools and resources, making their development process more efficient and productive.

### Category

Frontend

### Sub Category

Cat - OpenSource  ,   SubCat - Projects 

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by parma1111**
## Fixes Issue

Closes #575 #620 #658 #659 

## Changes proposed

The project aims to provide developers with a centralized platform to access a wide range of frontend tools and resources, making their development process more efficient and productive.

## Screenshots

Screenshot (811)
1) Improved-description
Screenshot (812)
2) Added First Interaction
Screenshot (814)
3) Added Github's Maintainer Community 
Screenshot (815)
4) Added-OpenSauced
Screenshot (816)

## Note to reviewers

Any kind of feedback would be appreciated.

### CODE REVIEW

File: database/opensource/projects.json [hunk 1]
```diff
@@ -19,5 +19,12 @@
     "url": "https://github.com/asakatsuOrg/AsaKatsuProject",
     "category": "opensource",
     "subcategory": "projects"
+  },
+  {
+    "name": "Web-Resources-Project",
+    "description": "'The project aims to provide developers with a centralized platform to access a wide range of front-end tools and resources, making their development process more efficient and productive.",
```
Comment (rupali-codes) [inline database/opensource/projects.json:25]:
```suggestion
    "description": "The project aims to provide developers with a centralized platform to access a wide range of front-end tools and resources, making their development process more efficient and productive.",
```

Comment (parma1111) [reply database/opensource/projects.json]: done @rupali-codes

**Review by rupali-codes — APPROVED** [review]
looks good to me

**Comment by rupali-codes** [comment]
@parma1111 are you a gssoc contributor?

**Comment by parma1111** [comment]
> @parma1111 are you a gssoc contributor?

> NO, @rupali-codes

**Review by CBID2 — APPROVED** [review]
Well done @parma1111! You've made great progress since the last PR. Well done! 😊LGTM!

**Comment by parma1111** [comment]
> Well done @parma1111! You've made great progress since the last PR. Well done! 😊LGTM!

> Thank you @CBID2 , I have learnt a lot of things from you.😀😀

**Comment by parma1111** [comment]
when I push the code

**1st way** 
I make the branch
git add .
git commit -m "message"
git push origin branch name

**2nd way**
I don't make branch
git add .
git commit -m "message"
git push origin old branch name

**3rd way**
When I get the error "git pull.... or fetch"
git pull
git merge origin/branch_name
git branch --set-upstream-to=origin/branch_name  branch_name
git push

_So there is any mistake in pushing way or it takes time to merge the PR without any reason._

**Comment by CBID2** [comment]
> when I push the code
> 
> 
> 
> **1st way** 
> 
> I make the branch
> 
> git add .
> 
> git commit -m "message"
> 
> git push origin branch name
> 
> 
> 
> **2nd way**
> 
> I don't make branch
> 
> git add .
> 
> git commit -m "message"
> 
> git push origin old branch name
> 
> 
> 
> **3rd way**
> 
> When I get the error "git pull.... or fetch"
> 
> git pull
> 
> git merge origin/branch_name
> 
> git branch --set-upstream-to=origin/branch_name  branch_name
> 
> git push
> 
> 
> 
> **So there is any mistake in pushing way or it takes time to merge the PR without any reason.** @rupali-codes @CBID2 
> 
> 

Probably the latter

**Comment by rupali-codes** [comment]
@parma1111 please DON'T use _git add ._ it would you take you just 2 mins extra, but add files manually by using _git add file1_pat file2_path_

**Review by CBID2 — APPROVED** [review]
LGTM! 🥳👍🏾

**Comment by rupali-codes** [comment]
> > @parma1111 please DON'T use _git add ._ it would you take you just 2 mins extra, but add files manually by using _git add file1_pat file2_path_
> 
> > @rupali-codes may you explain it clearly and merge it also ("_git add file1_pat file2_path_"). I could not understand it.

Which commands are you using for staging your changes? is it _git add ._?

**Comment by parma1111** [comment]
> > > @parma1111 please DON'T use _git add ._ it would you take you just 2 mins extra, but add files manually by using _git add file1_pat file2_path_
> > 
> > 
> > > @rupali-codes may you explain it clearly and merge it also ("_git add file1_pat file2_path_"). I could not understand it.
> 
> Which commands are you using for staging your changes? is it _git add ._?

Yes

**Pull request merged**